.vscode/settings.json

```diff
@@ -6,5 +6,7 @@
   "[typescript]": {
     "editor.formatOnSave": false
   },
-  "files.eol": "\n"
+  "files.eol": "\n",
+  "editor.wordWrapColumn": 150,
+  "html.format.wrapLineLength": 0
```

Comment:
Please undo this change.  If that is a problem, then we should remove settings.json from the repo.